src/main/java/com/techsophy/tsf/runtime/form/controller/impl/FormAclControllerImpl.java

```diff
@@ -30,7 +32,8 @@ public ApiResponse<FormAclDto> saveFormAcl(FormAclDto formAclDto) throws JsonPro
     }
 
     @Override
-    public ApiResponse<FormAclDto> getFormAcl(String formId) throws JsonProcessingException {
+    @Transactional
```

Comment:
we dont need transactional on get

Reply:
ok vaibhav removing transaction on get api